beacon-chain/rpc/apimiddleware/custom_handlers.go

```diff
@@ -15,92 +15,10 @@ import (
 	"github.com/prysmaticlabs/prysm/v4/api/gateway/apimiddleware"
 	"github.com/prysmaticlabs/prysm/v4/api/grpc"
 	"github.com/prysmaticlabs/prysm/v4/beacon-chain/rpc/eth/events"
-	http2 "github.com/prysmaticlabs/prysm/v4/network/http"
 	"github.com/prysmaticlabs/prysm/v4/runtime/version"
 	"github.com/r3labs/sse/v2"
 )
 
-type sszConfig struct {
-	fileName     string
-	responseJson SszResponse
-}
-
-func handleProduceBlockSSZ(m *apimiddleware.ApiProxyMiddleware, endpoint apimiddleware.Endpoint, w http.ResponseWriter, req *http.Request) (handled bool) {
-	config := sszConfig{
-		fileName:     "produce_beacon_block.ssz",
-		responseJson: &VersionedSSZResponseJson{},
```

Comment:
You can remove this type `VersionedSSZResponseJson` too since it's no longer in use, same goes for `SszResponseJson`

Reply:
I removed all leftover SSZ code

Reply:
I still see these being used in `beacon-chain/rpc/apimiddleware/structs.go`